backend/typescript/types.ts

```diff
@@ -25,6 +25,17 @@ export type ApplicationDashboardDTO = {
   applicationId: number;
 };
 
+export type ApplicationDashboardInput = {
```

Comment:
Is this type just supposed to be the `ApplicationDashboardDTO` without `applicationId`? If so, we should use `Omit<ApplicationDashboardDTO, "applicationId">` as the definition.

Reply:
Fixed!